### PR TITLE
Feat/v1.8.0 beta 2

### PR DESCRIPTION
**Pull request by abhimutant**

### CODE REVIEW

**Comment by abhimutant** [comment]
ok-to-test era

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (12.8) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (13.4) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (13.5) is below threshold (50). Kindly add more acceptance tests.

File: website/docs/d/ndb_clone.html.markdown [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+## Argument Reference
+
+* `clone_id`: (Optional) Clone id
+* `clone_name`: (Optional) Clone Name
+* `filters`: (Optional) Fetchs info based on filter
+
+### filters
+* `detailed`: (Optional) Default is false
```
Comment (bhati-pradeep) [inline website/docs/d/ndb_clone.html.markdown]: Explain their significance

File: website/docs/r/ndb_clone.html.markdown [hunk 2]
```diff
@@ -0,0 +1,122 @@
+layout: "nutanix"
+page_title: "NUTANIX: nutanix_ndb_clone"
+sidebar_current: "docs-nutanix-resource-ndb-clone"
+description: |-
+  This operation submits a request to perform clone of the database instance in Nutanix database service (NDB).
+---
+
+# nutanix_ndb_clone
+
+Provides a resource to perform the clone for database instance based on the input parameters.
```
Comment (bhati-pradeep) [inline website/docs/r/ndb_clone.html.markdown]: "clone of "

File: website/docs/r/ndb_clone.html.markdown [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+---
+
+# nutanix_ndb_clone
+
+Provides a resource to perform the clone for database instance based on the input parameters. 
+
+## Example Usage
+
+```hcl
+## resource for ndb_clone with Point in time given time machine name
```
Comment (bhati-pradeep) [inline website/docs/r/ndb_clone.html.markdown]: resource for cloning using point in time

File: website/docs/d/ndb_clone.html.markdown (hunk 1 above)
Comment (abhimutant) [reply website/docs/d/ndb_clone.html.markdown]: Done.

**Review by siddharth-kulshrestha — APPROVED** [review]

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (13.5) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (13.5) is below threshold (50). Kindly add more acceptance tests.

**Pull request merged**